redisenterprise/datadog_checks/redisenterprise/__about__.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.1'
+__version__ = '0.4.8'
```

Comment:
@sarah-witt because we ended up having to bump a lot of times to get this working correctly on production to do a lot of testing.  Even with a small change I'm ending up testing on production servers and thus ended up bumping the version a lot.

Reply:
Could you bump to 1.0.0 instead?

Reply:
@sarah-witt  bumped to 1.0.0!  Thanks